source/_patterns/01-atoms/button/_button-dropdown-split.twig

```diff
@@ -0,0 +1,18 @@
+{#
+  Split Button Dropdown!
+#}
+
+  {% include '@atoms/button/_button.twig' with {
+    button_color: dropdown_color,
+    button_text: dropdown_text,
+  } %}
+
+  {% embed '@atoms/button/_button.twig' with {
+    other_classes: 'dropdown-toggle dropdown-toggle-split',
```

Comment:
Rename all of these instances of 'other_classes' and 'other_attributes' to something more specific to the pattern like 'dropdown_other_classes'

Reply:
What are we using the `other_classes` for? It'd be nice if we can keep it BEM-y for clarity. `component_element_modifier_classes` or the like. This gets sticky for sure, but in naming it should be easy for people walking in to know what to change.

Reply:
Other_classes: isn't actually a class, its a variable to allow users to put in other classes that I don't allow for in the template itself.  I ran into several situations where I needed to throw more stuff on that wouldn't make sense in the template like on source/_patterns/01-atoms/button/_button-dropdown.twig.